components/roadmap-grid/Roadmap.module.css

```diff
@@ -48,7 +48,7 @@
   z-index: 4;
```

Comment:
The changes in this file should not be in this PR at all. It's not obvious until parsing through the entire thing that you didn't actually make any non-style/formatting changes.